### ISSUE TITLE
"lrupsc_2_2" does not exist

### ISSUE THREAD

**Issue by PandaxZA** [kyleconroy/sqlc#1341]
### Version

1.11.0

### What happened?

Seeing an error when making a basic query:

`ERROR: prepared statement "lrupsc_2_2" does not exist (SQLSTATE 26000)`

The query.sql looks the following:
```
-- name: GetContract :one
SELECT * FROM contracts
where fund_ticker = $1;
```

### What operating system are you using?

macOS

### What database engines are you using?

PostgreSQL

### What type of code are you generating?

Go

**Comment by PandaxZA** [comment]
To add here, this is occurring in a linux docker image

**Comment by kyleconroy** [comment]
@PandaxZA Can you add a set of reproduction steps?

**Comment by PandaxZA** [comment]
@kyleconroy - completely inconsistent. I've been logging it out, as a scheduled query, and it occurs about twice every three hours. I simplified the sql query to just ```SELECT * FROM table WHERE column = $1```

The error is logged out as:
```err=pq: unnamed prepared statement does not exist```

Happy to try out pgx and report back

**Comment by kyleconroy** [comment]
I don't think this is an issue with sqlc; this instead sounds like a bug in lib/pq.

**Comment by PandaxZA** [comment]
Thanks @kyleconroy , confirmed. We can close this issue. 

For anyone else experiencing similar issues. It's due to connection pooling, and should be addressed in https://github.com/lib/pq/pull/759

As a workaround, you can add ```binary_parameters=yes``` to the connection string
